pkg/handler/handler_v2.go

```diff
@@ -253,3 +253,13 @@ func (h *Handler) GetApiV2ServersServerIDConsolesessions(w http.ResponseWriter,
 
 	util.WriteJSONResponse(w, r, http.StatusOK, result)
 }
+
+// todo(schristoff): implement these
+func (h *Handler) GetApiV2Filestorage(w http.ResponseWriter, r *http.Request, params openapi.GetApiV2FilestorageParams) {
```

Comment:
Happy to make a sub-branch and sub-feat-branch PR and then merge into here once we get approvals if we don't want to merge todos like this.